docs/gateway-api/gateway-api-v1.md

```diff
@@ -228,22 +250,31 @@ AKO accepts the following Gateway configuration for this release:
   2. Gateway MUST NOT contain protocols other than HTTP or HTTPS.
   3. Gateway MUST contain a hostname. Hostname as `*` is not supported and `*.domain` is supported.
   4. Gateway MUST NOT contain TLS modes other than `Terminate`.
+  5. If the Secret specified in Gateway-> listeners-> TLSConfig does not exist or is in a namespace other than the Gateway namespace, AKO Gateway implementation will attach `SystemDefaultCert` to the Gateway. Once Secret is created in the same namespace as that of Gateway, the `certRef` will be updated accordingly.
+  6. Gateway CAN have multiple listeners with same/overlapping hostname. 
+  7. Two Gateways MUST NOT have listeners with same/overlapping hostname.
+  8. `Gateway-> listeners-> allowedRoutes-> namespaces-> from` with value `selector` is not supported.
+  9. `Gateway-> listeners-> allowedRoutes-> namespaces-> selector` is not supported.
+  
 
 #### HTTPRoute Limitations
 
 AKO accepts the following HTTPRoute configuration for this release:
 
   1. HTTPRoute MUST contain at least one parent reference.
   2. HTTPRoute MUST NOT contain `*` as hostname.
-  3. HTTPRoute MUST contain at least one hostname match with parent Gateway
+  3. HTTPRoute MUST NOT contain `*` in hostname.
+  4. HTTPRoute MUST contain at least one hostname match with parent Gateway
+  5. If HTTPRoute specify multiple hostnames and at least one has matching Gateway listener hostname and other one does not match with any of the Gateway listener hostname, HTTPRoute will be accepted, but traffic will only be allowed via valid hostname.
```

Comment:
Why does we require to publish this? This should be standard behaviour across GW implementation. Isn't it?

Reply:
Removed it